src/client/js/Utils/InterpreterManager.js

```diff
@@ -45,77 +45,105 @@ define([
         return pluginResult;
     };
 
+    InterpreterManager.prototype.loadConfigDialog = function (metadata, callback) {
+        if (!metadata.configWidget) {
+            callback(null, PluginConfigDialog);
+        } else {
+            requirejs([metadata.configWidget],
+                function (CustomConfigDialog) {
+                    callback(null, CustomConfigDialog);
+                },
+                function (err) {
+                    callback(err);
```

Comment:
I think it's better fail hard here. The plugin might be unusable without the custom configuration.